app/models/competition.rb

```diff
@@ -950,6 +950,10 @@ def can_edit_registration_fees?
   end
 
   def registration_opened?
+    use_wca_registration? && !cancelled? && !registration_not_yet_opened?
+  end
+
+  def registration_is_open?
     use_wca_registration? && !cancelled? && !registration_not_yet_opened? && !registration_past?
```

Comment:
Also, unrelated: If you introduce `registration_opened?` you can just reuse that here:

```suggestion
    registration_opened? && !registration_past?
```